Makefile

```diff
@@ -42,41 +46,49 @@ ifeq ($(OS_NAME),fedora)
 	DOCKERFILE = rpm.dockerfile
 	PKG_TYPE = rpm
 endif
-RELEASE_IMAGE_NAME = promscale-extension-pkg:$(EXT_VERSION)-$(OS_NAME)$(OS_VERSION)-$(PG_VER)
-RELEASE_FILE_NAME = promscale_extension-$(EXT_VERSION).$(PG_VER).$(OS_NAME)$(OS_VERSION).$(ARCH).$(PKG_TYPE)
-TESTER_NAME = pg$(PG_VERSION)-$(OS_NAME)$(OS_VERSION)
+RELEASE_IMAGE_NAME = promscale-extension-pkg:$(EXT_VERSION)-$(OS_NAME)$(OS_VERSION)-$(PG_RELEASE_VER)
+RELEASE_FILE_NAME = promscale_extension-$(EXT_VERSION).$(PG_RELEASE_VER).$(OS_NAME)$(OS_VERSION).$(ARCH).$(PKG_TYPE)
+TESTER_NAME = $(PG_RELEASE_VER)-$(OS_NAME)$(OS_VERSION)
 
 .PHONY: help
 help:
-	@echo "promscale_extension $(EXT_VERSION) (pg$(PG_VERSION))"
+	@echo "promscale_extension $(EXT_VERSION)"
 	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: build
 build: ## Build the extension
-	cargo build --release --features pg${PG_VERSION} $(EXTRA_RUST_ARGS)
-	cargo pgx schema -f pg${PG_VERSION} --release
+	cargo build --release --features pg${PG_BUILD_VERSION} $(EXTRA_RUST_ARGS)
+	cargo pgx schema -f pg${PG_BUILD_VERSION} --release
 
 .PHONY: clean
 clean: ## Clean up latest build
 	cargo clean
 
+.PHONY: prepackage
+prepackage: ## Used in docker build to improve the build caching
+	cargo pgx schema pg${PG_BUILD_VERSION} --out sql/promscale--${EXT_VERSION}.sql
```

Comment:
does this line help cache any dependencies? can we get rid of line 69 and 71 and just keep `cargo pgx package`?

Reply:
It does help to cache dependencies, I've left it as-is but added a comment to clarify.